src/main.cpp

```diff
@@ -718,13 +717,14 @@ bool AcceptToMemoryPoolWorker(CTxMemPool &pool,
 
     // Rather not work on nonstandard transactions (unless -testnet/-regtest)
     std::string reason;
+    const CChainParams &chainparams = Params();
```

Comment:
is the comment above, "// Rather not work on non-stand.ard ..."  still accurate?

Reply:
Nothing has changed in the behavior of this function.  

I suppose the comment has never been entirely accurate since you can override the testnet/regtest defaults to enable refusal of nonstandard transactions.  But I think that its ok for the comment to speak about common use / default behavior.